tests/xbmcaddon.py

```diff
@@ -6,7 +6,6 @@
 # pylint: disable=invalid-name
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from xbmc import getLocalizedString
```

Comment:
I'd probably have kept this import here, maybe using import ... as ..., but I'm fine with this, too.
Once restructuring is done, I think we can move imports back to the top, then we can have another thought about this.

Reply:
I did this to avoid a circular import in the CLI test (`make test-run`)

Reply:
Ok, in the long run this should probably be solved differently, but that shouldn't affect this PR. I'll merge this in ~2h if there are no objections.